vision/automl/edge_container_predict/README.md

```diff
@@ -0,0 +1,50 @@
+# AutoML Vision Edge Container Prediction
+
+This is an example to show how to predict with AutoML Vision Edge Containers.
+The test (freeze_meta_graph_lib_test.py) shows an automatical way to run the
+prediction.
+
+If you want to try the test manually with a sample model, please install
+[gsutil tools](https://cloud.google.com/storage/docs/gsutil_install) and
+[Docker CE](https://docs.docker.com/install/) first, and then follow the steps
+below. All the following instructions with commands assume you are in this
+folder.
+
++   Step 1. Pull the Docker image.
+
+```bash
+$ CPU_DOCKER_GCS_PATH=gcr.io/automl-vision-ondevice/gcloud-container-1.12.0:latest
```

Comment:
say a few words explaining what this docker image is for.

Reply:
added.

Reply:
Thanks.  Could you also add the link to the source (e.g. the Dockerfile) into this comment?

Reply:
added a link to tfserving docker hub.